.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,4 @@
+<!-- Keep the progress checklist.  -->
 ## Progress
```

Comment:
I'm curious on who would be responsible for updating it. Would it be the contributor of the request, or a terra core contributor? It could be hard for a contributor to constantly have to keep the checklist up to date I envision.